### DISCUSSION TITLE
MONAI Deploy vs MONAI Label

### DISCUSSION THREAD

**Discussion by junxiant** [Project-MONAI/monai-deploy-app-sdk#434] [General]
Hello,

I was taking a look at MONAI Deploy sdk, I have seen the youtube videos and the tutorials, but I'm still quite unclear how it compares to MONAILabel?

If I already have a MONAILabel server running, would I still want to be using MONAI Deploy? Since the MONAI Label server allows for API calling/inference to the model.

**Comment by MMelQin** [comment]
Thanks for the question @junxiant.

It is worthwhile to look at the phases of AI in practice: label, train, deploy. MONAI Deploy App SDK was purposely created to accelerate building containerized inference application with integration with medical imaging networks, which can be hosted on MONAI Deploy or other workflow platforms.   https://monai.io/
